daemon/loop.go

```diff
@@ -149,6 +150,52 @@ func (d *Daemon) pullAndSync(logger log.Logger) {
 		}); err != nil {
 			logger.Log("err", err)
 		}
+
+		// Find notes in revisions.
+		for i, rev := range revisions {
+			// Skip last revision (since we requested HEAD~1 above)
+			if i == len(revisions)-1 {
+				continue
+			}
+			n, err := working.GetNote(rev)
+			if err != nil {
+				logger.Log("err", errors.Wrap(err, "loading notes from repo; possibly no notes"))
```

Comment:
Oh I see, it can get an error if the note ref doesn't exist? Fair enough; maybe we can catch that earlier and present it as "no note and no error". Not necessarily in this PR.